src/view.js

```diff
@@ -151,14 +316,429 @@ deepAssign($view, {
+	 * @parent can.view.static
+	 * @description Render a template.
+	 * @signature `can.view.render(template[, callback])`
+	 * @param {String|Object} view The path of the view template or a view object.
+	 * @param {Function} [callback] A function executed after the template has been processed.
+	 * @return {Function|can.Deferred} A renderer function to be called with data and helpers
+	 * or a Deferred that resolves to a renderer function.
+	 *
+	 * @signature `can.view.render(template, data[, [helpers,] callback])`
+	 * @param {String|Object} view The path of the view template or a view object.
+	 * @param {Object} [data] The data to populate the template with.
+	 * @param {Object.<String, function>} [helpers] Helper methods referenced in the template.
+	 * @param {Function} [callback] A function executed after the template has been processed.
+	 * @param {NodeList} nodelist parent nodelist to register partial template contents with
+	 * @return {String|can.Deferred} The template with interpolated data in string form
+	 * or a Deferred that resolves to the template with interpolated data.
+	 *
+	 * @body
+	 * `can.view.render(view, [data], [helpers], callback)` returns the rendered markup produced by the corresponding template
+	 * engine as String. If you pass a deferred object in as data, render returns
+	 * a deferred resolving to the rendered markup.
+	 *
+	 * `can.view.render` is commonly used for sub-templates.
+	 *
+	 * ## Example
+	 *
+	 * _welcome.ejs_ looks like:
+	 *
+	 *     <h1>Hello <%= hello %></h1>
+	 *
+	 * Render it to a string like:
+	 *
+	 *     can.view.render("welcome.ejs",{hello: "world"})
+	 *       //-> <h1>Hello world</h1>
+	 *
+	 * ## Use as a Subtemplate
+	 *
+	 * If you have a template like:
+	 *
+	 *     <ul>
+	 *       <% list(items, function(item){ %>
+	 *         <%== can.view.render("item.ejs",item) %>
+	 *       <% }) %>
+	 *     </ul>
+	 *
+	 * ## Using renderer functions
+	 *
+	 * If you only pass the view path, `can.view will return a renderer function that can be called with
+	 * the data to render:
+	 *
+	 *     var renderer = can.view.render("welcome.ejs");
+	 *     // Do some more things
+	 *     renderer({hello: "world"}) // -> Document Fragment
+	 *
+	 */
+	//call `renderAs` with a hardcoded string, as view.render
+	// always operates against resolved template files or hardcoded strings
+	render: function (view, data, helpers, callback, nodelist) {
+		return can.view.renderAs("string",view, data, helpers, callback, nodelist);
+	},
+
+	// ##### renderTo
+	//
+	/**
+	 * @hide
+	 * @function renderTo
+	 * @param {String} format
+	 * @param {Function} renderer
+	 * @param data
+	 * @param {Object} helpers helper methods for this template
+	 * @param {NodeList} nodelist parent nodelist to register partial template contents with
+	 * @return {*}
+	 */
+	renderTo: function(format, renderer, data, helpers, nodelist){
+		if (format === "string") {
+			if (renderer.renderAsString) {
```

Comment:
This will check for the actual string renderer now, which is provided by `ejs` here:

https://github.com/canjs/can-ejs/pull/67/files#diff-d693f1cbebb75316de0f7e3e21837a56R23